src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -110,24 +113,27 @@ limitations under the License.
     <item name="Video" value="0x0D"/>
   </enum>
 
-  <enum name="ContentLaunchStatusEnum" type="enum8">
+  <enum name="StatusEnum" type="enum8">
     <cluster code="0x050a"/>
-    <item name="success" value="0x00"/>
-    <item name="urlNotAvailable" value="0x01"/>
-    <item name="authFailed" value="0x02"/>
+    <item name="Success" value="0x00"/>
+    <item name="URLNotAvailable" value="0x01"/>
+    <item name="AuthFailed" value="0x02"/>
+    <item name="TextTrackNotAvailable" value="0x03"/>
+    <item name="AudioTrackNotAvailable" value="0x04"/>
   </enum>
 
   <struct name="ParameterStruct">
     <cluster code="0x050a"/>
     <item name="Type" type="ParameterEnum"/>
-    <item name="Value" type="char_string"/>
+    <item name="Value" type="char_string" length="1024" />
     <item name="ExternalIDList" type="AdditionalInfoStruct" array="true" optional="true"/>
   </struct>
 
-  <bitmap name="SupportedStreamingProtocol" type="bitmap32">
+  <bitmap name="SupportedProtocolsBitmap" type="bitmap32">
```

Comment:
Does this need a compat shim in `src/app/CompatEnumNames.h`?

Reply:
Added. I was not aware of that file.